src/database/models/guild.js

```diff
@@ -1,9 +1,15 @@
 import mongoose from 'mongoose';
 
 const GuildSchema = new mongoose.Schema({
-  id: { type: String }, //ID of the guild
-  registeredAt: { type: Number, default: Date.now() },
-
+  id: {
+    type: String,
```

Comment:
This `id` prop although not introduced in this PR might be confusing with the autogenerated `_id` prop which can also be accessed using `id` getter sometimes. The type of `id` is ObjectID which is not compatible with String.

I leave it up to you to determine relevance 👍🏼